.circleci/config.yml

```diff
@@ -501,8 +496,8 @@ jobs:
     steps:
       - checkout
       - add_ssh_keys:
-           fingerprints:
-             - "3b:cd:47:bf:57:9c:e5:36:b0:4d:5f:12:5e:d3:b3:3e"
+          fingerprints:
+            - "3b:cd:47:bf:57:9c:e5:36:b0:4d:5f:12:5e:d3:b3:3e"
```

Comment:
There are 5 times we reuse key fingerprint. Probably it would make sense to make a custom command with that.

Reply:
I just moved this into the install-mbx-ci command since that's what it's associated with (see https://github.com/mapbox/mapbox-maps-ios/pull/424)